packages/extensions-sdk/src/cli/commands/validators/check-license.ts

```diff
@@ -0,0 +1,33 @@
+import fse from 'fs-extra';
+import { type Ora } from 'ora';
+import type { Report } from '../../types.js';
+
+const checkLicense = {
+	name: 'license',
+	handler: async (spinner: Ora, reports: Array<Report>) => {
+		spinner.text = 'Check for LICENSE';
+
+		if (!(await fse.pathExists(`${process.cwd()}/LICENSE`))) {
+			spinner.fail();
+			const message = 'No LICENSE.md';
```

Comment:
If we are checking for the `LICENSE` file only this should be the error message as well (without the `.md`)
```suggestion
			const message = 'No file named LICENSE found';
```